pkg/cableengine/syncer/syncer.go

```diff
@@ -124,7 +124,7 @@ func (gs *GatewaySyncer) syncGatewayStatusSafe() {
 	} else if err != nil {
 		utilruntime.HandleError(fmt.Errorf("error getting existing Gateway: %w", err))
 		return
-	} else if !reflect.DeepEqual(gatewayObj.Status, existingGw.Status) {
+	} else if (gatewayObj.Status.HAStatus == v1.HAStatusPassive) || !reflect.DeepEqual(gatewayObj.Status, existingGw.Status) {
```

Comment:
We really should update the timestamp for the active GW as well. I was preparing a commit that also adds a unit test but this PR is fine for a quick fix for backport to 0.13.

Reply:
Ack, 10x Thom.
I checked the 2 GWs use case on my env with this fix - looks OK.